docs/releases.rst

```diff
@@ -8,7 +8,8 @@ tmt-1.32
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The hardware specification for :ref:`/spec/hardware/disk` has been
-extended with the new ``model-name`` key.
+extended with the new ``model-name`` key. Users can provision beaker
+servers with a given model disk model using mrack plugin.
```

Comment:
A couple of stylistic notes:

* "Beaker" instead of "beaker",
* "guests" instead of "servers",
* "beaker" plugin instead of "mrack" plugin,
* and we can reference to the plugin as well:

```
... using :ref:`/spec/plans/provision/beaker` plugin.
```